users/luescher/setups/librispeech/pipeline_base_args.py

```diff
@@ -41,9 +44,22 @@ def get_init_args(
     mfcc_extra_args: Optional[Dict] = None,
     gt_normalization: bool = True,
     gt_options_extra_args: Optional[Dict] = None,
+    tying_type: str = "global",
+    nonword_phones: str = "",
     tdp_transition: Tuple[
         Union[float, str], Union[float, str], Union[float, str], Union[float, str]
-    ] = (3.0, 0.0, 30.0, 0.0),
+    ] = (3.0, 0.0, "infinity", 0.0),
```

Comment:
Having everything configurable as parameter is very different to my approach, that the baseline only gives a fixed setting and you should copy and alter the whole file.

maybe we should discuss this to have it consistent across the different baselines.

Reply:
> users/luescher

I think we can fix as soon as we move to `common` dir :)